### PR TITLE
api: search by instagram and facebook

### PR DESCRIPTION
**Pull request by razbakov**
TInputProfile should search profiles by WeDance username, name, instagram and facebook. 

This component is used now in Organiser and Artists fields in Event Editor.

### CODE REVIEW

**Review by best-lucky1030 — COMMENTED** [review]
Yes, it works.

**Comment by harshbrahmxatriya** [comment]
Please add a description including 'how to test it' part.

**Comment by razbakov** [comment]
@harshbrahmxatriya @Stefanacef I updated description

**Comment by razbakov** [comment]
Will be part of #150 as its a coupled issue

**Review by amanuela97 — APPROVED** [review]

File: components/TInput/TInputArray.vue [hunk 1]
```diff
@@ -31,9 +31,18 @@ export default {
     },
   },
   watch: {
+    value(val) {
+      console.log('value changed', val)
```
[the remark this answers is not in the document]
Comment (razbakov) [reply components/TInput/TInputArray.vue]: good catch, but that file is not part of that PR 😅

**Pull request merged**